e2e/main_test.go

```diff
@@ -23,12 +23,19 @@ func TestMain(m *testing.M) {
 
 	flag.Parse()
 
+	logFile, err := os.Create("ramentest.log")
```

Comment:
"e2e.log"?

I'm not sure this should be the default. Logging to stdout is very easy to use and our log is not very large.

Maybe add --logfile option?

```console
./run.sh -logfile e2e.log
```

Reply:
We can also always log to file regardless of stdout/stderr. This makes it easier to use the log after running other commands with lot of output.

If we always log to file, maybe we can use much less detailed log to stderr, maybe only the normal test output.

I also want to always keep drenv log in standard location to avoid the issue of not having logs to debug issues like this:
https://cloud-native.slack.com/archives/C043N6ZFV9S/p1732122420443799